apps/gitness/src/pages/pipeline-edit/context/PipelineStudioDataProvider.tsx

```diff
@@ -1,22 +1,22 @@
 import React, { createContext, useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import { useParams } from 'react-router-dom'
 import { isEmpty, isUndefined } from 'lodash-es'
 import { Skeleton } from '@harnessio/canary'
 import { useYamlEditorContext } from '@harnessio/yaml-editor'
-// import CreatePipeline from 'pages/CreatePipeline/CreatePipeline'
-import { countProblems, monacoMarkers2Problems } from '../utils/problems-utils'
-import type { YamlProblemSeverity } from '../types/types'
-import type { InlineActionArgsType } from '../utils/inline-actions'
-import { deleteItemInArray, injectItemInArray, updateItemInArray } from '../utils/yaml-utils'
-import { Problem } from '@harnessio/playground'
-import { TypesPlugin } from '../types/api-types'
 import {
   OpenapiGetContentOutput,
   TypesPipeline,
   TypesSignature,
   useFindPipelineQuery,
   useGetContentQuery
 } from '@harnessio/code-service-client'
-import { useParams } from 'react-router-dom'
+import { Problem } from '@harnessio/playground'
+// import CreatePipeline from 'pages/CreatePipeline/CreatePipeline'
```

Comment:
do we need this import?

Reply:
Probably added by Srdjan. @srdjan-harness pls check if we can remove it.